dags/main_summary.py

```diff
@@ -382,6 +380,7 @@
             "scalar_parent_aushelper_websense_reg_version",
             "scalar_parent_dom_contentprocess_troubled_due_to_memory_sum",
             "--replace",
+            "DATE_FORMAT(profile_creation_date, 'yyyy-MM-dd 00:00:00') AS profile_creation_date",
```

Comment:
Is this necessary? This is already a string in the raw table, isn't it?

Reply:
discussed in slack, this will be a string in the raw table